pkg/promutil/promutil_test.go

```diff
@@ -13,29 +14,23 @@ func TestMustCounterValue(t *testing.T) {
 	})
 
 	result := MustCounterValue(counter)
-	if result != 0.0 {
-		t.Errorf("Initial counter value should be 0.0, got %v", result)
-	}
+	require.Equal(t, 0.0, result, "Initial counter value should be 0.0")
```

Comment:
Refactoring these tests to use `require` to bring them in line with the rest of the codebase.